static/js/navigation.js

```diff
@@ -2,5 +2,15 @@
 $(".go-back").click(function(event) {
 	event.preventDefault();
 	const previousPagePath = $(event.target).data("previous-page-path");
-	AP.navigator.go('addonmodule', { moduleKey: previousPagePath });
+	const closeOnBack = $(event.target).data("close-on-back");
+
+	if (closeOnBack) {
+		window.close();
+	} else {
+		if (previousPagePath) {
+			AP.navigator.go('addonmodule', { moduleKey: previousPagePath });
+		} else {
+			history.back();
+		}
```

Comment:
/nit reduce nesting, could return early or change the `else { if {` to a `else if`

```
if (closeOnBack) {
 window.close();
 return;
}
if (previousPagePath) {
 AP.navigator.go('addonmodule', { moduleKey: previousPagePath });
 return;
}
history.back();
```

```
if (closeOnBack) {
 window.close();
} else if (previousPagePath) {
 AP.navigator.go('addonmodule', { moduleKey: previousPagePath });
} else {
 history.back();
}
```

Reply:
Updated by using `if-elseif-else`